packages/edit-navigation/src/store/transform.js

```diff
@@ -46,7 +46,8 @@ export function blockToMenuItem(
 	blockPosition,
 	menuId
 ) {
-	menuItem = omit( menuItem, 'menus', 'meta', '_links' );
+	const { menus, meta, _links, ...restMenuItem } = menuItem;
+	menuItem = restMenuItem;
```

Comment:
Looks good 👍 

FYI this package is likely to be removed anyway.

Reply:
Thanks! Sounds good - removing unused code is always a step in the right direction 🚀